harvard-university-of-bath.csl

```diff
@@ -527,7 +527,7 @@
           <group prefix=" " delimiter=" ">
             <choose>
               <if variable="editor">
-                <text term="in" text-case="capitalize-first" suffix=":"/>
+                <text term="in" text-case="capitalize-first" suffix=": "/>
```

Comment:
Could you revert this please? This gets displayed incorrectly in the preview for technical reasons, but in all relevant software, the delimiter from line 527 above will apply here. Sorry about the confusion, we're trying to get that fixed.

Reply:
Sure. Thanks for the explanation. When I saw the preview I was worried the toolchain I use for testing locally had been doing it wrong all this time...

Reply:
> Sorry about the confusion, we're trying to get that fixed.

@adam3smith, did we have a ticket for this?